_posts/2022-08-27-ownership-in-an-age-of-microservices.md

```diff
@@ -7,63 +7,63 @@ tags: [Management, Microservices, Tech, Resilience]
 
 * Internal tickets
 * Internal open-source (my preferred)
 * Trusted outsider
 * Tour of duty
 * Embedded expert
 
-These are all explained very well in a post [patterns of cross-team collaboration](https://blog.thepete.net/blog/2021/06/17/patterns-of-cross-team-collaboration/). I personally like team ownership acting as stewards of the long term vision and domain of services, influencing changes via an internal open-source model. The teams in these cases are similar to the primary contributors to open source projects.
+These are all explained very well in a post [patterns of cross-team collaboration](https://blog.thepete.net/blog/2021/06/17/patterns-of-cross-team-collaboration/). I personally like team ownership, acting as stewards of the long term vision and domain of services, and influencing changes via an internal open-source model. The team, in this model, plays a similar role to that of the primary contributors in an open source project.
 
-# Ownership, strong ownership takes time
+# Strong ownership takes time
 
-In the decomposed microservices world, I personally believe that having a team with a long history of knowledge building and maintaining an app will lead to better outcomes. The institutional knowledge of operating the service, feeds back into the development practices of the system. Learning how to avoid risk, anticipate needs, and more quickly and flexibly change and update the system over time. It also helps with communications, especially at times of incident response when everyone knows the team that owns an app and is confident that the team has deep knowledge of the system and will be able to support it. A stable relationship between business domain, services, and team brings clarity and shared knowledge of where responsibilities and vision for the domain belongs. In my last post, I explain how reorgs and diverging team names across systems causes [organizational drift](/management/2022/08/22/hr-org-dev-org) and confusion.
+In a decomposed, microservices ecosystem I personally believe that having a team with a long history of knowledge-building and maintaining an app will lead to better outcomes. The institutional knowledge of operating the service feeds back into the development practices of the system (team?). Over time, the team learns how to mitigate risk, anticipate needs, and more quickly and flexibly change and evolve the system. Strong team ownership also helps with communication, especially during incident response as everyone knows which team owns a given app, which adds confidence that the team also has deep knowledge of the system and will be able to support it. A stable relationship between business domain, services, and team brings clarity and shared knowledge of where responsibilities and vision for the domain belongs. In my last post, I will discuss how reorgs and diverging team names across systems causes [organizational drift](/management/2022/08/22/hr-org-dev-org) and confusion.
```

Comment:
> feeds back into the development practices of the system

did you mean to say team here? I wasn't sure.

Reply:
I was thinking more of the code that makes it's way into the system, but since I said development practices, I agree team seems more clear to me as well